packages/block-editor/src/components/use-block-sync/index.js

```diff
@@ -49,6 +61,10 @@ import { useRegistry } from '@wordpress/data';
  *                                is used to initalize the block-editor store
  *                                and for resetting the blocks to incoming
  *                                changes like undo.
+ * @param {string} props.entityId An ID which should identify the "thing" to which
+ *                                the blocks belong. This is used to keep track
+ *                                of times when two different components need to
+ *                                sync the same blocks.
  * @param {Object} props.selectionStart The selection start vlaue from the
  *                                controlling component.
  * @param {Object} props.selectionEnd The selection end vlaue from the
```

Comment:
Not related to this PR, but the lack of space between parameter name and description probably messes up type info a bit.

Reply:
Actually, I just realized that there's no issue here. I thought the `Start` and `End` were part of the descriptions, rather than the parameter names.